cmd/system-probe/modules/network_tracer.go

```diff
@@ -248,7 +248,9 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 	})
 
 	httpMux.HandleFunc("/debug/usm_telemetry", telemetry.Handler)
-	httpMux.HandleFunc("/debug/usm_traced_programs", usm.TracedProgramsEndpoint)
+	httpMux.HandleFunc("/debug/usm/tls/traced_programs", usm.TracedProgramsEndpoint)
+	httpMux.HandleFunc("/debug/usm/tls/attach-pid", usm.AttachPIDEndpoint)
+	httpMux.HandleFunc("/debug/usm/tls/detach-pid", usm.DetachPIDEndpoint)
```

Comment:
nit: as mentioned IRL I don't think we need to make this change TLS-specific

Reply:
What do you mean?
Today, only tls is scoped to a pid, I cannot attach or detach from plain http or plain kafka

Reply:
I would just avoid mentioning TLS. 
```
debug/usm/traced_programs
debug/usm/attach_pid
``` 
etc

Reply:
done